### PR TITLE
Use new filesystem key manager

### PR DESCRIPTION
**Pull request by sethvargo**
Depends on https://github.com/google/exposure-notifications-server/pull/957

**Release Note**

```release-note
Recommend filesystem key manager for local development
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/google/exposure-notifications-verification-server
 
 go 1.14
 
+replace github.com/google/exposure-notifications-server => ../exposure-notifications-server
```
Comment (sethvargo) [inline go.mod:5]: Will need to update this after merging https://github.com/google/exposure-notifications-server/pull/957

File: pkg/database/database.go [hunk 2]
```diff
@@ -89,25 +89,6 @@ func (c *Config) Load(ctx context.Context) (*Database, error) {
 		logger.Errorf("key manager does not support the SigningKeyManager interface, falling back to single verification signing key")
 	}
 
-	// If the key manager is in-memory, accept the key as a base64-encoded
```
Comment (sethvargo) [inline pkg/database/database.go:92]: I'm glad to get rid of this - it was... hacky.

File: go.mod (hunk 1 above)
Comment (mikehelmick) [reply go.mod:5]: pls update to v0.7.0

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**